### PR TITLE
Title Load

### PR DESCRIPTION
**Pull request by Zemurin**

### CODE REVIEW

File: CK3toEU4/Source/CK3World/Titles/Title.h [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	std::string article;															// "the ". Not always present.
+	std::string historyGovernment;											// Unclear why this is history. Maybe further governments override it.
+	date creationDate;															// Unclear. Ranges to 9999.1.1, probably is PDX alternative for "bool isCreated";
+	std::pair<int, std::shared_ptr<CoatOfArms>> coa;					// This is dejure flag but not defacto.
+	std::pair<int, std::shared_ptr<Title>> dfLiege;						// defacto liege title (d_kalyani)
+	std::pair<int, std::shared_ptr<Title>> djLiege;						// dejure liege title (d_rattapadi)
+	std::pair<int, std::shared_ptr<Character>> holder;					// Holding character
+	std::map<int, std::shared_ptr<Title>> dfVassals;					// defacto vassals, not in save, manually linked post-loading
+	std::map<int, std::shared_ptr<Title>> djVassals;					// dejure vassals (for all except baronies and titulars)
+	std::vector<std::pair<int, std::shared_ptr<Character>>> heirs; // Order of heirs is unclear so we're keeping them ordered and using fist if able.
```
Comment (Idhrendur) [inline CK3toEU4/Source/CK3World/Titles/Title.h:51]: fist -> first

File: CK3toEU4/Source/CK3World/Characters/Character.h [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  private:
+	void registerKeys();
+
+	int charID = 0;
+	std::string name;
+	std::pair<int, std::string> culture;
+	std::pair<int, std::string> religion;
+	std::map<int, std::string> traits;
+
```
Comment (Idhrendur) [inline CK3toEU4/Source/CK3World/Characters/Character.h:33]: Were this many blank lines intentional?

**Review by Idhrendur — APPROVED** [review]
Just aesthetic things.

File: CK3toEU4/Source/CK3World/Characters/Character.h (hunk 2 above)
Comment (Zemurin) [reply CK3toEU4/Source/CK3World/Characters/Character.h:33]: Placeholder file, barely clanged.

**Pull request merged**